### PR TITLE
BUG Fix incorrect namespaces

### PR DESCRIPTION
**Pull request by tractorcow**
Update circleci to use php 7.1

Thanks @adrexia for noticing the issue at https://github.com/Firesphere/silverstripe-graphql-jwt/commit/621d312473ffcf568beaac41f1fb962dbd7f1194#r34697262

### CODE REVIEW

**Comment by tractorcow** [comment]
@Firesphere know of any good docker lamp images? I'll try a few I've found via a quick search.

**Comment by tractorcow** [comment]
Hm I don't think I can trigger a rebuild.

**Comment by Firesphere** [comment]
I think the config for CircleCI needs updating?

**Comment by tractorcow** [comment]
I did update it though in the last commit. I guess it's not like travis, and a new push triggers rebuilds?

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: brettt89/silverstripe-web
+      - image: mattrayner/lamp:latest-1804
```
Comment (Firesphere) [inline .circleci/config.yml:9]: Why the switch of docker image?

File: .circleci/composer.json.test [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "firesphere/graphql-jwt-test",
   "description": "Composer file for CircleCI Tests",
   "require": {
-    "php": ">=5.6.0",
+    "php": ">=7.1.0",
```
Comment (Firesphere) [inline .circleci/composer.json.test:5]: I'd even say 7.2, but I know that's not an option for lower 4.x releases

File: .circleci/config.yml [hunk 3]
```diff
@@ -19,13 +19,14 @@ jobs:
       - image: circleci/mysql:5.7
         environment:
           - MYSQL_USER=root
-          - MYSQL_ROOT_PASSWORD=ubuntu
+          - MYSQL_ROOT_PASSWORD=
```
Comment (Firesphere) [inline .circleci/config.yml]: This makes me uncomfortable, no root PWD?

File: .circleci/config.yml [hunk 4]
```diff
@@ -19,13 +19,14 @@ jobs:
...
           - MYSQL_DATABASE=circle_test
           - MYSQL_HOST=127.0.0.1
 
     working_directory: ~/var/www
 
     steps:
+      - run: sudo composer self-update
```
Comment (Firesphere) [inline .circleci/config.yml]: Is a self-update really necessary?

File: .gitignore [hunk 5]
```diff
@@ -1,3 +1,5 @@
 /vendor
 /mysite
-/resources
\ No newline at end of file
+/resources
+composer.lock
+/resources
```
Comment (Firesphere) [inline .gitignore]: duplicate resources?

File: src/Authentication/AnonymousUserFactory.php [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1);
```
Comment (Firesphere) [inline src/Authentication/AnonymousUserFactory.php:1]: Yash!

**Review by Firesphere — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (tractorcow) [reply .circleci/config.yml:9]: Beacuse that image is like php 5.6 or something.

File: .circleci/config.yml (hunk 3 above)
Comment (tractorcow) [reply .circleci/config.yml]: I don't think it needs one for a temporary image.

File: .circleci/config.yml (hunk 4 above)
Comment (tractorcow) [reply .circleci/config.yml]: Images can have outdated packages.

File: .gitignore (hunk 5 above)
Comment (tractorcow) [reply .gitignore]: Ah true.

**Comment by tractorcow** [comment]
Sorry @Firesphere I tried to do my due dilligence and fix your circle-ci, but I gave up after it was too hard.

**Comment by tractorcow** [comment]
@Firesphere can we just merge this anyway? I'm not going to be able to get the tests passing without a lot of work. The tests are already broken.

**Comment by tractorcow** [comment]
I've made an attempt to fix your tests, but to be honest, can you do this instead? I just want to fix the namespace, not repair your broken circle CI.

**Comment by tractorcow** [comment]
I can't even use the old mysql docker image. Sorry I'm out of time to spend. =(

File: .circleci/config.yml (hunk 1 above)
Comment (Firesphere) [reply .circleci/config.yml:9]: Brett also has a 7.x image now, brettt89/silverstripe-web:{php-version}

**Comment by Firesphere** [comment]
Could you please squash? I'll have a look at the test errors later when I have time and energy.

**Comment by tractorcow** [comment]
@Firesphere i've squashed all the docker updates into one commit, and left the main update in another.

**Pull request merged**